src/TextUI/Configuration/Cli/Builder.php

```diff
@@ -55,6 +55,7 @@ final class Builder
         'covers=',
         'uses=',
         'help',
+        'check-scope',
```

Comment:
@sebastianbergmann 

Not sure about the order here (and in other places). 

What do you think?